app/utils/generic.py

```diff
@@ -17,7 +17,7 @@
 )
 from requests import post as requests_post
 
-from app.models.dialogue import show_fatal_error, show_warning
+import app.views.dialogue as dialogue
```

Comment:
Minor: for consistency, is there a reason to not import the methods that are used here rather than `dialogue` itself? Other files import the methods

Reply:
Was a little hack to prevent circular dependency I found. I forgot with what it was, I think generic. If both modules don't use each other on just loading, then you avoid circular dep error if you import the module itself instead of its components.